src/core/public/index.ts

```diff
@@ -183,7 +183,7 @@ export { MountPoint, UnmountCallback, PublicUiSettingsParams } from './types';
  * navigation in the generated docs until there's a fix for
  * https://github.com/Microsoft/web-build-tools/issues/1237
  */
-export interface CoreSetup<TPluginsStart extends object = object> {
+export interface CoreSetup<TPluginsStart extends object = object, TStart = unknown> {
```

Comment:
We may want to add some `@typeParam` docs for these now. It's not exactly obvious what should be passed in here.

Reply:
Did not know about `@typeParam`, will add.

Reply:
Added, however this annotation doesn't seem to be reflected in the generated doc